components/src/index.ts

```diff
@@ -0,0 +1,110 @@
+export { AppFrame } from '@pluralsight/ps-design-system-appframe'
```

Comment:
We are missing some of our exports.

If you look in most `packages/**/src/index.ts` files, there are exports for `css`, `vars` and `js`. I want users of the stable grouped package or the latest individual packages to get the same functionality. I think we can find a way to do that.

I think we can drop `css` because we're transitioning to vanilla css.

But I think `vars` and `js` should get exported too.  

0. What do you think: should we export those? 
1. What is the naming convention to avoid collision in this single namespace? Not sure yet.
2. Is that this PR? Probably not, because I don't have an answer to number 1. If not this PR, would you mind carding it?

Reply:
When I originally started this refactor using 2a approach these exports seemed most reasonable. They now seem even more so. I actually think this will create too many exports if we export vars and js. Also I say this factor makes a good argument for keeping the statics. As for css no there is no need to export that js as we're moving to vanilla css.

Reply:
1. How do we know we have "too many" exports? What's the threshold for that standard?

2. I feel like if we cut off the `js` exports we lose a good amount of functionality that we could expose. Do you sense this?

3. If we lean on statics, why have some exports like `themeNames` at all? One reason is that statics will bloat bundle size. Things will be attached to each other and unable to be tree-shaken. Named exports will solve that. Do you see the same problem?